go/db/sqlite.go

```diff
@@ -0,0 +1,109 @@
+package db
```

Comment:
Generally I would suggest not to place both sql scripts and db-related go code in the same directory as it is logically separated. So maybe we can create /go/app/db directory (with the same package name)